src/title-dropdown/title-dropdown.component.html

```diff
@@ -1,7 +1,9 @@
-<div dropdown>
-  <button class="btn title-dropdown-toggle" dropdownToggle>
+<div dropdown [ngClass]="{disableCursorClass: schema && schema['x_editor_disabled']}">
+  <div [ngClass]="{disableEventsClass: schema && schema['x_editor_disabled']}">
+    <button class="btn title-dropdown-toggle" dropdownToggle>
```

Comment:
but how i will know if it is disabled? I mean i ll pass the schema and apply the dropdown isDisabled instead of the class you mean?